src/DynamoCore/Graph/Nodes/CodeBlockNode.cs

```diff
@@ -245,18 +245,20 @@ public void SetCodeContent(string newCode, ElementResolver workspaceElementResol
 
                 var inportConnections = new OrderedDictionary();
                 var outportConnections = new OrderedDictionary();
+                var inportPins = new OrderedDictionary();
+                var outportPins = new OrderedDictionary();
```

Comment:
Not sure i understand what inportPins are?

Reply:
Added summaries. Basically for the CodeBlockNode, when there is an undo/redo action things are deleted and reconstructed. That includes connectors and ports. They reconstruct INports and OUTports. These lists store collections of ConnectorPins for each.